tor/helpers/youtube.py

```diff
@@ -24,8 +23,8 @@ def get_yt_video_id(url):
       Invalid:
         'youtu.be/watch?v=_lOT2p_FCvA',
     """
-    # initial version: http://stackoverflow.com/a/7936523/617185 \
-    #    by Mikhail Kashkin (http://stackoverflow.com/users/85739/mikhail-kashkin)
+    # initial version: http://stackoverflow.com/a/7936523/617185 \ by Mikhail
```

Comment:
In this case, I think we can remove the `\` before "by Mikhail". Not entirely certain why it was there to begin with, TBH.